### PR TITLE
Add docs page, fix download page and minor style fixes

### PR DESCRIPTION
**Pull request by khangmach**
Add download page and minor style fixes

### CODE REVIEW

**Comment by khangmach** [comment]
Note that I don't have all the download links in there yet :) Could use some help on that

**Comment by jtamiace** [comment]
Is it possible to have the Windows tab pre-selected on page load? Wasn't clear that the download button was referring to version until I clicked the different OS tabs (but maybe that has something to do with not having all the download links yet)

screen shot 2017-12-01 at 2 29 27 pm

**Comment by khangmach** [comment]
@jtamiace nice catch. Fixed!

File: fle_site/apps/main/templates/main/download.html [hunk 1]
```diff
@@ -0,0 +1,112 @@
...
+                <a class="nav-link" data-toggle="tab" href="#PyPi" role="tab">PyPi</a>
+            </li>
+        </ul>
+
+        <!-- Tab panes -->
+        <div class="tab-content">
+            <div class="tab-pane active" id="windows" role="tabpanel">
+                <a class="download-btn" href="#">DOWNLOAD</a><br><br><br>
+                <p>Compatible with Windows 7+</p>
+                <a href="#">Installation instructions for Windows</a>
```
Comment (radinamatic) [inline fle_site/apps/main/templates/main/download.html]: http://kolibri.readthedocs.io/en/develop/install.html#windows

File: fle_site/apps/main/templates/main/download.html [hunk 2]
```diff
@@ -0,0 +1,112 @@
...
+            </div>
+            <div class="tab-pane" id="android" role="tabpanel">
+                <a class="download-btn" href="#">DOWNLOAD</a><br><br><br>
+                <p>Compatible with Android 4+</p>
+                <a href="#">Installation instructions for Android</a>
+            </div>
+            <div class="tab-pane" id="debian" role="tabpanel">
+                <a class="download-btn" href="#">DOWNLOAD</a><br><br><br>
+                <p>Compatible with Debian 1+</p>
+                <a href="#">Installation instructions for Debian</a>
```
Comment (radinamatic) [inline fle_site/apps/main/templates/main/download.html]: http://kolibri.readthedocs.io/en/develop/install.html#linux

**Comment by jtamiace** [comment]
Also assume that this is to be pasted in later? :P 

screen shot 2017-12-01 at 2 33 52 pm

**Comment by khangmach** [comment]
@jtamiace Yeah I think Lauren's providing a license statement soon. Or they'll literally have agree to consume all of those pastries before they download

File: fle_site/apps/main/templates/main/download.html [hunk 3]
```diff
@@ -0,0 +1,112 @@
...
+            </div>
+            <div class="tab-pane" id="PyPi" role="tabpanel">
+                <a class="download-btn" href="#">DOWNLOAD</a><br><br><br>
+                <p>Compatible with PyPi 1+</p>
+                <a href="#">Installation instructions for PyPi</a>
```
Comment (radinamatic) [inline fle_site/apps/main/templates/main/download.html]: http://kolibri.readthedocs.io/en/develop/install.html#advanced-installation-options

**Comment by jtamiace** [comment]
> Or they'll literally have agree to consume all of those pastries before they download

_@jtamiace approves this pull request_

File: fle_site/apps/main/templates/main/download.html [hunk 4]
```diff
@@ -0,0 +1,112 @@
...
+        <div class="tab-content">
+            <div class="tab-pane active" id="windows" role="tabpanel">
+                <a class="download-btn" href="#">DOWNLOAD</a><br><br><br>
+                <p>Compatible with Windows 7+</p>
+                <a href="#">Installation instructions for Windows</a>
+            </div>
+            <div class="tab-pane" id="android" role="tabpanel">
+                <a class="download-btn" href="#">DOWNLOAD</a><br><br><br>
+                <p>Compatible with Android 4+</p>
+                <a href="#">Installation instructions for Android</a>
```
Comment (radinamatic) [inline fle_site/apps/main/templates/main/download.html]: http://kolibri.readthedocs.io/en/develop/install.html#android

**Comment by khangmach** [comment]
@radinamatic should I even include the text "installation instructions for X" if that link would be the same as the readthedocs links on each download button?

**Comment by radinamatic** [comment]
@khangmach Not sure what are you suggesting... :thinking: 
Didn't we decide to put the link to [`Latest release`](https://github.com/learningequality/kolibri/releases/latest) inside the download button?

Considering there are going to be at least 4 links with installation instructions, putting the most appropriate context inside the link text is recommended.

**Comment by khangmach** [comment]
@radinamatic I put all the links you gave me above in each download button. However, for the "installation instruction" hyperlinks (there are 4 of them currently, each following the download button), where should those be linked to?

**Comment by indirectlylit** [comment]
> Note that I don't have all the download links in there yet :) Could use some help on that

You can use these links for now: https://github.com/learningequality/kolibri/releases/tag/v0.7.0-beta5

**Comment by radinamatic** [comment]
It was my understanding that the download button would carry the link to the [`Latest release`](https://github.com/learningequality/kolibri/releases/tag/v0.7.0-beta5) (more precise one provided by @indirectlylit above), and the link for each OS installation instructions would lead to the actual heading inside the docs.

File: fle_site/apps/main/templates/main/download.html [hunk 5]
```diff
@@ -0,0 +1,114 @@
...
+{% block bodycontents %}
+
+
+    <!-- Header -->
+    <div id="body">
+
+        <div class="container-fluid download-container">
+            <div class="row">
+                <div class="col-sm-6 col-md-6">
+                    <img src="{% static 'img/kickstarter/download-kolibri.png' %}" alt="download-box-image" class="center-block">
```
Comment (radinamatic) [inline fle_site/apps/main/templates/main/download.html]: This image looks to me purely decorative, you can leave the alternative text empty `alt=""`.

File: fle_site/apps/main/templates/main/download.html [hunk 6]
```diff
@@ -35,14 +35,14 @@
         <div class="container-fluid download-container">
             <div class="row">
                 <div class="col-sm-6 col-md-6">
-                    <img src="{% static 'img/kickstarter/download-kolibri.png' %}" alt="download-box-image" class="center-block">
+                    <img src="{% static 'img/kickstarter/download-kolibri.png' %}" class="center-block">
```
Comment (radinamatic) [inline fle_site/apps/main/templates/main/download.html]: Here it was about leaving `alt` **empty**, not leaving it **out**... :wink:

**Comment by indirectlylit** [comment]
looking good. I'll merge in and we can continue to iterate on it.

@khangmach make sure you switch over to the `develop` branch if you're going to make further commits. Let me know if you need help

**Pull request merged**